charts/datadog/templates/cluster-agent-scc.yaml

```diff
@@ -17,6 +17,8 @@ allowHostIPC: false
 allowHostPID: false
 allowPrivilegeEscalation: false
 allowPrivilegedContainer: false
+allowedCapabilities: null
+defaultAddCapabilities: null
```

Comment:
This 2 lines should not be added back

Reply:
ahhhh I thought that you had added those. It was a bit weird with your merge earlier today ;)

Reply:
@clamoriniere - fixed this now 👍